src/components/notification/z-cookiebar/index.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { newSpecPage } from "@stencil/core/testing";
+
+import { ZCookiebar } from "./index";
+
+describe("Suite test ZCookiebar", () => {
+  it("Test render ZCookiebar base", async () => {
+
+    const page = await newSpecPage({
+      components: [ZCookiebar],
+      html: `  <z-cookiebar
+                  cookiepolicyurl="https://my.zanichelli.it/privacy#Cookie"
+                  hide="false"></z-cookiebar>`
+    });
+
+    expect(page.root).toEqualHtml(`
+    <z-cookiebar
+      cookiepolicyurl="https://my.zanichelli.it/privacy#Cookie"
+      hide="false">
+        <mock:shadow-root>
+          <z-candybar class="">
+            <div class="content" slot="content">
+              <div>
+                <h2>Su questo sito usiamo i cookie.</h2>
```

Comment:
Il testo non va passato tramite prop (in previsione della parte di traduzione) ?